File: WebApp/Scripts/app/model/ParamD.js
Ext.define('WEBAPP.model.ParamD', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'GRP_CODE', type: 'string' },
        { name: 'DATA_SEQ', type: 'int' },
        { name: 'DATA_NAME', type: 'string' },
        { name: 'DATA_VALUE', type: 'string' },
        { name: 'DATA_DESC', type: 'string' },
        { name: 'DATA_REMARK', type: 'string' }
    ]
});

File: WebApp/Models/PARAM_D.cs
﻿namespace WebApp.Models
{
    public class PARAM_D : JCLib.Mvc.BaseModel
    {
        public string GRP_CODE { get; set; }
        public int DATA_SEQ_O { get; set; }
        public int DATA_SEQ { get; set; }
        public string DATA_NAME { get; set; }
        public string DATA_VALUE { get; set; }
        public string DATA_DESC { get; set; }
        public string DATA_REMARK { get; set; }
    }
}